src/Config/Install.props

```diff
@@ -3,8 +3,12 @@
   <ImportGroup Label="PropertySheets" Condition=" Exists('$(SolutionDir)Config/user_local.props') ">
     <Import Project="$(SolutionDir)Config/user_local.props" />
   </ImportGroup>
+  <PropertyGroup>
+	<OutputPath Condition=" '$(OutputPath)' == '' ">$(SolutionDir)..\bin\$(Platform)\$(Configuration)\$(REVIT_VERSION)</OutputPath>
```

Comment:
Earlier the MSI was built in its default OutputDir, but now you are changing it to the common bin, which is not right. Usually, the installer will be made at the end when we have built all the branches of Revit. Similarly, we don't need to define TestOutputPath because it's not needed for installer project. 
Install.props is supposed to be used by installer only.